scene/3d/audio_stream_player_3d.cpp

```diff
@@ -35,6 +35,99 @@
 #include "scene/3d/listener.h"
 #include "scene/main/viewport.h"
 
+// Based on "A Novel Multichannel Panning Method for Standard and Arbitrary Loudspeaker Configurations" by Ramy Sadek and Chris Kyriakakis (2004)
+// Speaker-Placement Correction Amplitude Panning (SPCAP)
+class Spcap {
```

Comment:
I think the class could be named `SPCAP` all uppercase, since it's an acronym. We use PascalCase for class names, keeping acronyms capitalized (e.g. `ARVRServer`).